frontend/Store.js

```diff
@@ -614,6 +620,17 @@ class Store extends EventEmitter {
       this._nodesByName = this._nodesByName.set(node.get('name'), this._nodesByName.get(node.get('name')).delete(id));
     }
   }
+
+  _setGetterValue(data: DataType) {
+    var node = this._nodes.get(data.id);
+    var props = node.get('props');
+    var last = data.path.length - 1;
+    var propObj = data.path.slice(0, last).reduce((obj_, attr) => obj_ ? obj_[attr] : null, props);
+    propObj[data.path[last]] = data.value;
+    var newProps = assign({}, props);
```

Comment:
Seems like tangential unrelated issue (worth raising). Why is this specific to getters (and not just any prop mutations)?